packages/angular_devkit/build_webpack/src/webpack/index2.ts

```diff
@@ -0,0 +1,90 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import { BuilderContext, BuilderOutput, createBuilder } from '@angular-devkit/architect/src/index2';
+import { getSystemPath, json, normalize, resolve } from '@angular-devkit/core';
+import { Observable, from, of } from 'rxjs';
+import { switchMap } from 'rxjs/operators';
+import * as webpack from 'webpack';
+import { ArchitectPlugin } from '../plugins/architect';
+import { Schema as RealWebpackBuilderSchema } from './schema';
+
+const webpackMerge = require('webpack-merge');
+
+export type WebpackBuilderSchema = json.JsonObject & RealWebpackBuilderSchema;
+
+export interface WebpackLoggingCallback {
+  (stats: webpack.Stats, config: webpack.Configuration): void;
+}
+export interface WebpackFactory {
+  (config: webpack.Configuration): Observable<webpack.Compiler>;
+}
+
+
+export function runWebpack(
```

Comment:
The index2_spec_large.ts are the tests moved from the previous builder. This is as much of an entry point as any public/protected class members from the previous builder.

Reply:
Just as a FYI, I'm planning to do a coverage at one point before release, and add tests for missing pieces where it makes sense.

Reply:
Since builders can no longer be directly augmented or have their behavior otherwise adjusted, utility functions such as this and the ones in the second PR are going to be used extensively in third-party builders.  I think extensive test coverage would be prudent in this case to not only ensure functionality but to also prevent future unintentional breaking changes.